City-of-Helsinki/kukkuu-admin
# KK-580 | Add browser tests for messages feature

## Description

In this PR I am adding browser based tests for the messaging feature.

## Context

Because this feature is close to being finished, I have added happy path tests that take inspiration from the user story.

I don't actually send the message in any of these tests, because sent messages can't be deleted.

I've configured a custom project for these test to run in, in order to have better assurances that the test data is not tampered with by accident.

[KK-580](https://helsinkisolutionoffice.atlassian.net/browse/KK-580)

## How Has This Been Tested?

This PR contains one fixture with two happy path tests. I've tested that they work locally as well as against the test version of kukkuu admin.

## Manual Testing Instructions for Reviewers

Ensure that your environment provides access to the required variables:
- `BROWSER_TESTS_UID`
- `BROWSER_TESTS_PWD`

These can be found from the secrets repo in the kukkuu-admin e2e keys file.

1. Run local tests with `yarn test:browser`
1. Run tests again staging server `yarn test:browser:ci`
